docs/reference/cluster/stats.asciidoc

```diff
@@ -1477,6 +1477,18 @@ The API returns the following response:
           ...
         }
       },
+      "index_pressure": {
+        "memory": {
+            "current": {
+                ...
+            },
+            "total": {
+                ...
```

Comment:
I think we should spell out the actual fields here. There aren't that many, and it's super-useful to see them in the docs.

Reply:
I add fields but the test fail I will check later